kong/init.lua

```diff
@@ -179,6 +179,32 @@ do
 end
 
 
+local is_data_plane
+local is_control_plane
+local is_dbless
+local is_dp_worker_process
+do
+  is_data_plane = function(config)
+    return config.role == "data_plane"
+  end
+
+
+  is_control_plane = function(config)
+    return config.role == "control_plane"
+  end
+
+
+  is_dbless = function(config)
+    return config.database == "off"
+  end
+
+
+  is_dp_worker_process = function()
+    return process.type() == "privileged agent"
+  end
```

Comment:
```suggestion
  is_dp_worker_process = function(config)
    return is_data_plane(config)
           and process.type() == "privileged agent"
  end
```

Reply:
No, `dp_worker` is not only for role `data_plane`, but also for role `traditional`.

Reply:
According the logic before, only data_plane will enable privileged process, so I think we need not do it.

Reply:
#9860 removed privileged process, so I also removed this function.